### DISCUSSION TITLE
Best way to fetch updated IP Whitelist

### DISCUSSION THREAD

**Discussion by yogesh-sharma-gtg** [codemagic-ci-cd/codemagic-docs#2569] [Ideas]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

Authorization issue due to changes in IP Whitelists.
We currently have Private repos which are used for builds and therefore we require IP whitelisting.
The document here is helpful - https://docs.codemagic.io/partials/firewall/
However, we need to visit the page to get the list of IPs for whitelisting.

### Expected results

Better way to pull the IPs required to be whitelisted.
Is there an API or a DNS record we might can use to fetch the new set of IPs whenever there is an update?

### Actual results

Proactively visit IP whitelist webpage.

**Comment by mikhail-tokarev** [comment]
thank you for this idea! we will update our API to include this information

**Comment by mikhail-tokarev** [comment]
Hi @yogesh-sharma-gtg 
just a quick update that this information now available via https://codemagic.io/api/v3/meta API

you can read more here https://codemagic.io/api/v3/schema#/operations/ApiV3MetaMeta
let me know in case you have any questions. Thank you!

**Reply by yogesh-sharma-gtg** [reply]
Thank you so much for the update and notification. Really appreciate it.
